b2sdk/transfer/outbound/copy_source.py

```diff
@@ -80,3 +81,9 @@ def get_copy_source_range(self, relative_offset, range_length):
             source_file_info=self.source_file_info,
             source_content_type=self.source_content_type
         )
+
+    def get_large_file_sha1(self):
+        if self.offset or self.length:
```

Comment:
what if `self.length` spans the full file?

Reply:
If `self.length` would match the full file length, we could copy the `LARGE_FILE_SHA1` value too.  

Unfortunately, the size of the copy source file is not available here.  It's also not available in `Bucket.copy`, which is the only caller within the SDK.  The only way to get it would be by making an extra request to the API.  

So far in my changes I assumed that the `LARGE_FILE_SHA1` will only be set in the basic scenarios and in a best-effort manner.  Is it OK to skip the corner case you spotted?  What's your thoughts?

Reply:
It's ok - we can always add it later if we find out that it's a common case